components/kyma-environment-broker/internal/process/input/input.go

```diff
@@ -376,8 +377,15 @@ func (r *RuntimeInput) adjustRuntimeName() error {
 }
 
 func (r *RuntimeInput) configureOIDC() error {
-	if r.provisioningParameters.Parameters.OIDC == nil {
-		// TODO: read and use default values
+	if !r.provisioningParameters.Parameters.OIDC.IsProvided() {
+		r.provisionRuntimeInput.ClusterConfig.GardenerConfig.OidcConfig = &gqlschema.OIDCConfigInput{
+			ClientID:       r.oidcDefaultValues.ClientID,
+			GroupsClaim:    r.oidcDefaultValues.GroupsClaim,
+			IssuerURL:      r.oidcDefaultValues.IssuerURL,
+			SigningAlgs:    r.oidcDefaultValues.SigningAlgs,
+			UsernameClaim:  r.oidcDefaultValues.UsernameClaim,
+			UsernamePrefix: r.oidcDefaultValues.UsernamePrefix,
+		}
```

Comment:
also upgradeShootInput should be updated, let's do it when the conflict will be resolved in this PR: https://github.com/kyma-project/control-plane/pull/795

Reply:
ok